src/Symfony/Component/Process/Process.php

```diff
@@ -1154,7 +1154,7 @@ public function getInput()
      *
      * This content will be passed to the underlying process standard input.
      *
-     * @param resource|scalar|\Traversable|null $input The content
+     * @param int|float|string|bool|resource|\Traversable|null $input The content
```

Comment:
👎 on the removal of `scalar`

Reply:
@GrahamCampbell please review discussion ;)